ts/src/pro/kucoin.ts

```diff
@@ -189,14 +190,32 @@ export default class kucoin extends kucoinRest {
         await this.loadMarkets ();
         symbols = this.marketSymbols (symbols);
         let messageHash = 'tickers';
+        const messageHashes = [];
+        const topics = [];
         if (symbols !== undefined) {
             messageHash = 'tickers::' + symbols.join (',');
+            for (let i = 0; i < symbols.length; i++) {
+                const symbol = symbols[i];
+                messageHashes.push ('ticker:' + symbol);
+                const market = this.market (symbol);
+                topics.push ('/market/ticker:' + market['id']);
```

Comment:
The Topic value here should be loaded from the config if was overridden by the user because it can be `snapshot or `ticker` 

I'll create a pull request to fix this If you are in

Reply:
@ali-master sorry, please open a pr that reads that from params and options using `handleOptionAndParams`